src/moneypenny/errors.py

```diff
@@ -0,0 +1,45 @@
+"""Helper globals and errors for Moneypenny.
+"""
+
+namespace_file = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
+primary_container_image = "library/alpine:latest"
+dossier_file = "/opt/lsst/software/moneypenny/config/dossier/dossier.yaml"
+M = "/opt/lsst/software/moneypenny/config/M/m.yaml"
+quips = "/opt/lsst/software/moneypenny/config/quips/quips.txt"
+moneypenny_timeout = 300
```

Comment:
These seem to reiterate the constants from the config? Not sure why they're here.

Nit: For other projects, we've called the module collecting the exceptions `moneypenny.exceptions`.

Reply:
constants removed, errors->exceptions